invenio_records_resources/resources/errors.py

```diff
@@ -102,50 +103,52 @@ class ErrorHandlersMixin:
         QuerystringValidationError: create_error_handler(
             HTTPJSONException(
                 code=400,
-                description="Invalid querystring parameters.",
+                description=_("Invalid querystring parameters."),
```

Comment:
Maybe we skip marking the error messages for now until we decide on the approach?